lib/conditions/index.js

```diff
@@ -0,0 +1,50 @@
+const express = require('express');
+const logger = require('../logger').policy;
+const schemas = require('../schemas');
+const predefined = require('./predefined');
+const conditions = {};
+
+function register ({name, handler, schema}) {
+  let validate;
```

Comment:
ToDo: Change this function so it does not throw an exception. If it's returning a boolean, it means it's caller responsibility whether to throw an exception or not.